docs/apache-airflow-providers-amazon/connections/aws.rst

```diff
@@ -146,6 +146,18 @@ Snippet to create Connection and convert to URI
     os.environ[env_key] = conn_uri
     print(conn.test_connection())
 
+
+  .. warning:: When using airflow CLI there might be a need to add ``@`` when:
+
+    - login
+    - password,
+    - host
+    - port
```

Comment:
Why does only the password have a comma attached? Does this mean something I don’t get?

Reply:
👍 good find